sycl/plugins/unified_runtime/ur/adapters/level_zero/adapter.cpp

```diff
@@ -180,10 +180,9 @@ UR_APIEXPORT ur_result_t UR_APICALL urAdapterGetLastError(
                    ///< error code will be stored.
 ) {
   std::ignore = Adapter;
-  std::ignore = Message;
   std::ignore = Error;
-  urPrint("[UR][L0] %s function not implemented!\n", __FUNCTION__);
-  return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
+  *Message = ErrorMessage;
```

Comment:
thanks @JackAKirk . L0 implementation for getLastError is being done here https://github.com/intel/llvm/pull/10784. There's a discussion here on how it should behave. i'm planning on following on that, so feel free to address the hip and cpu adapters only on this patch, or to address also L0 based on those comments. Either way works for me.

Reply:
Sure, L0 impl here is already consistent with those comments (all backends have the same impl).

Reply:
@jandres742 do you want me to remove the l0 implementation from this PR, or are you happy with it?

Thanks

Reply:
@JackAKirk : implementation here is good, thanks!

Reply:
OK thanks!